atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -345,7 +345,11 @@ def get_logs(self):
             logfile = NamedTemporaryFile(prefix=self.build_id,
                                          suffix=".log",
                                          mode='wb')
-            logfile.write(logs.encode('UTF-8'))
+            try:
+                logfile.write(logs)
+            except UnicodeEncodeError:
```

Comment:
We can just `except (TypeError, UnicodeEncodeError):` our way around it I expect.